x-pack/test/functional/apps/monitoring/cluster/list.js

```diff
@@ -28,7 +28,7 @@ export default function({ getService, getPageObjects }) {
         await clusterList.assertDefaults();
       });
 
-      after(async () => {
+      after(async function clusterListingAfterAll() {
```

Comment:
Not sure what this change is for.  Can it be in a separate PR?

Reply:
Removing